source/directives/uiTreeNode.js

```diff
@@ -338,7 +341,8 @@
                   }
 
                   // Show the placeholder if it was hidden for nodrop-enabled and this is a new tree
-                  if (targetNode.$treeScope && !targetNode.$parent.nodropEnabled && !targetNode.$treeScope.nodropEnabled) {
+                  if (targetNode.$treeScope && !targetNode.$parent.nodropEnabled
+                    && !targetNode.$treeScope.nodropEnabled && placeElm.prop('tagName').toLowerCase() !== 'tr') {
```

Comment:
I don't think we need the extra check for a `tr` element once #631 is merged. Am I correct?
See https://github.com/angular-ui-tree/angular-ui-tree/pull/631/files#diff-2bd42e539aeba5277f65f2599dfc0338R342